### ISSUE TITLE
problem: no source code

### ISSUE THREAD

**Issue by gazhayes** [nanopool/XMR-stak#2]
Releasing binaries with no source code to show what you've changed is frowned upon. This is how people get their coins stolen.

**Comment by worthwhileindustries** [comment]
yeah and getting alerts from virus scanners is a little frowned upon also. https://github.com/nanopool/XMR-stak/issues/3

**Comment by psychocrypt** [comment]
@nanopool Please provide the source code of  the miner. xmr-stak is under GPL and you **must** provide the source code.
If not you will do a license violation.
